### PR TITLE
Configure Manuals Publisher application

### PR DESCRIPTION
**Pull request by Rosa-Fox**
- Part of the process to rename specialist-publisher to manuals-publisher

### CODE REVIEW

**Comment by alexmuller** [comment]
This looks good to me. I think it's worth removing all references to `nagios_memory_` in this pull request if we're not going to set it - we can add it later if the app ends up needing it.

**Comment by steventux** [comment]
@alexmuller / @deanwilson this PR has been amended, would one of you mind reviewing please.

**Comment by deanwilson** [comment]
It looks fine to me but as this is my first app add i'd like @alexmuller to have the final say.

File: modules/govuk/manifests/apps/manuals_publisher.pp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+#
+# [*redis_port*]
+#   Redis port for sidekiq.
+#   Default: 6379
+#
+class govuk::apps::manuals_publisher(
+  $port = 3064,
+  $asset_manager_bearer_token = undef,
+  $email_alert_api_bearer_token = undef,
+  $enabled = false,
```
Comment (steventux) [inline modules/govuk/manifests/apps/manuals_publisher.pp]: Should this be set to `true` initially?

**Pull request merged**